blocks/models/huggingface/loading.py

```diff
@@ -37,61 +28,13 @@ def _safe_load_training_pipeline(
             f"Training: Model loaded {config.task_type.value}: {PrintFormats.BOLD}{module.__class__.__name__ if isinstance(module, PreTrainedModel) else module}{PrintFormats.END}",
             level=1,
         )
-    except:
-        multi_line_print(
-            f"Training: Couldn't load {module} model or tokenizer. Skipping.", level=1
-        )
-        model, tokenizer = None, None
-
-    return model, tokenizer
-
-
-def _safe_load_inference_pipeline(
-    module: Union[str, PreTrainedModel],
-    config: HuggingfaceConfig,
-    tokenizer: Optional[PreTrainedTokenizerBase] = None,
-) -> Optional[Callable]:
-    try:
-        if tokenizer is not None:
-            loaded_pipeline = pipeline(
-                task=config.task_type.value,
-                model=module,
-                tokenizer=tokenizer,
-                device=device,
-            )
-        else:
-            loaded_pipeline = pipeline(
-                task=config.task_type.value, model=module, device=device
-            )
-
-        multi_line_print(
-            f"Inference: Pipeline loaded {config.task_type.value}: {PrintFormats.BOLD}{module.__class__.__name__ if isinstance(module, PreTrainedModel) else module}{PrintFormats.END}",
-            level=1,
-        )
+        return model, tokenizer
 
     except:
         multi_line_print(
-            f"Inference: Couldn't load {module} pipeline. Skipping.", level=1
+            f"Training: Couldn't load {module} model or tokenizer. Skipping.", level=1
```

Comment:
should I remove Training from here? I guess yes?

Reply:
Yes!